content/grants/app-urvault.md

```diff
@@ -0,0 +1,39 @@
++++
+
+title = "UrVault Apprenticeship"
+date = "2022-03-18"
+
+[taxonomies]
+grant_type = ["Apprenticeship"]
+grant_category = ["App Dev: Other"]
+
+[extra]
+image = ""
+description = "UrVault: Urbit Password Manager"
+reward = "$12000 in WSTR"
```

Comment:
```suggestion
reward = "$8000 in WSTR"
```

Apprenticeships are a [flat $8k in reward](https://urbit.org/grants/apprenticeships#funding) — the point of them is to learn, with payment happening as an added bonus on top of the opportunity to receive mentorship.